src/payment/strategies/stripev3/stripev3-payment-strategy.ts

```diff
@@ -1,281 +1,338 @@
         }
 
-        if (billingAddress) {
-            return {
-                ...stripeBillingDetails,
-                email: billingAddress.email,
-                name: `${billingAddress.firstName} ${billingAddress.lastName}`,
-            };
+        return stripev3;
+    }
+
+    private _getStripeElement(): StripeElement {
+        if (!this._stripeElement) {
+            throw new NotInitializedError(NotInitializedErrorType.PaymentNotInitialized);
         }
 
-        return {
-            name: 'Guest',
-        };
+        return this._stripeElement;
     }
 
-    private _mapStripeShippingDetails({ shippingAddress, customer }: { shippingAddress?: Address ; customer?: Customer } = {}): StripeShippingDetails {
-        const stripeShippingDetails = {
-            address: this._mapStripeShippingAddress(shippingAddress),
-        };
-
-        if (customer) {
-            return {
-                ...stripeShippingDetails,
-                name: `${customer.firstName} ${customer.lastName}`,
-            };
+    private _getStripeJs(): StripeV3Client {
+        if (!this._stripeV3Client) {
+            throw new NotInitializedError(NotInitializedErrorType.PaymentNotInitialized);
         }
 
-        if (shippingAddress) {
-            return {
-                ...stripeShippingDetails,
-                name: `${shippingAddress.firstName} ${shippingAddress.lastName}`,
-            };
+        return this._stripeV3Client;
+    }
+
+    private async _loadStripeJs(): Promise<StripeV3Client> {
+        if (this._stripeV3Client) { return this._stripeV3Client; }
+
+        const paymentMethod = this._store.getState().paymentMethods.getPaymentMethodOrThrow(this._getInitializeOptions().methodId);
+        const { initializationData: { stripePublishableKey, stripeConnectedAccount } } = paymentMethod;
+
+        return await this._stripeScriptLoader.load(
+            stripePublishableKey,
+            stripeConnectedAccount,
+            this._locale
+        );
+    }
+
+    private _mountElement({ options, containerId }: StripeV3PaymentInitializeOptions): Promise<StripeElement> {
+        const stripeElementType = this._getInitializeOptions().methodId as StripeElementType;
+
+        let stripeElement: StripeElement;
+
+        return new Promise((resolve, reject) => {
+            if (!this._stripeElements) {
+                this._stripeElements = this._getStripeJs().elements();
+            }
+
+            stripeElement = this._stripeElements.getElement(stripeElementType) || this._stripeElements.create(stripeElementType, options);
+
+            try {
+                stripeElement.mount(`#${containerId}`);
+            } catch (error) {
+                reject(new NotInitializedError(NotInitializedErrorType.PaymentNotInitialized));
+            }
+
+            resolve(stripeElement);
+        });
+    }
+
+    private _mapStripeAddress(address: Address | undefined, customer?: Customer, padding = false): unknown {
```

Comment:
instead of returning `unknown`, it might be worth having a `StripeBillingDetails ` type?

```
interface StripeBillingDetails {
  name: string;
  address: Partial<StripeAddress>;
}
```

Also, instead of having `address: Address | undefined`, maybe you can have `address?: Address`?